openapi_python_client/utils.py

```diff
@@ -66,10 +66,19 @@ def fix_reserved_words(value: str) -> str:
     return value
 
 
+def case_insensitive_snake_case(value: str) -> str:
```

Comment:
Should this be called `case_sensitive_snake_case` instead?

Reply:
Makes sense, fixed